src/lib/market-service/zerion/types.ts

```diff
@@ -0,0 +1,21 @@
+export type ZerionMarketData = {
+  price: number
+  total_supply: number
+  circulating_supply: number
+  fully_diluted_valuation: number
+  market_cap: number
+  changes: {
+    percent_1d: number
+    percent_30d: number
+    percent_90d: number
+    pecent_365d: number
+  }
+}
```

Comment:
preferably-blocking: `number | null` all fields
<img width="1496" alt="image" src="https://github.com/user-attachments/assets/fbd81dff-2ef2-47a7-a26f-ea7eaecc620e">

Reply:
Good call - addressed in https://github.com/shapeshift/web/pull/7489/commits/f5f012c79bf83ad8adad054e5cef596ec90910be & https://github.com/shapeshift/web/pull/7489/commits/d6b682c34c479bce0832e70dbf662efa3079f68a